datasets/util/ade20k/ade20k_calibration_files.txt

```diff
@@ -0,0 +1,500 @@
+ADE_train_00002043.jpg
```

Comment:
What is the license of this file, and where did it come from? That should be documented somewhere.